core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestrator.scala

```diff
@@ -198,7 +197,21 @@ class BillingProjectOrchestrator(ctx: RawlsRequestContext,
             )
           )
       }
-      jobControlId <- projectLifecycle.initiateDelete(projectName, ctx)
+      billingProfileId <- billingRepository.getBillingProfileId(projectName)
+
+      maybeGoogleProject = billingProfileId match {
```

Comment:
I am getting the CloudPlatform if possible to avoid making the GCP call to delete the parent Google project when we KNOW that the project is an Azure one. Arguably, I could remove this optimization and simplify the code.

Reply:
Done